faq/cohort-subs-guide.rst

```diff
@@ -0,0 +1,149 @@
+We can reccommend the following database resources for common data types:
+
+- For sensitive human nucleotide records and human clinical epidemiological data which requires controlled access, please
+  contact the `EGA (European Genome Phenome Archive) <https://ega-archive.org/>`_ to start a submission.
+- For expression data, or uncategorsied datasets, please use `ArrayExpress/BioStudies <https://www.ebi.ac.uk/biostudies/arrayexpress>`_
+
+After the datasets have been submitted in the appropriate databases, the required child Samples for linking can be created.
+The child samples will represent their relationship to the top-level Sample. Different samples can be used for different
+data types **and** for different time points. Please contact us if you have any doubts about setting up your sample structure.
+
+
+3. Link together the samples using BioSamples
+''''''''''''''''''''''''''''''''''''''''''''''
+
+Link your samples created from other EBI resources to the top-level sample using the ‘derived from’ curation
+on biosamples: https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_curation_object
+
+Link your samples created from other EBI resources to the top-level sample using the ‘derived from’ curation on
+BioSamples. The derived from relationship is used as follows, where the Source is the child Sample, and the Target is
+the top-level Sample:
+
+**Source sample** - *derived from* - **Target sample**
+
+**Child sample accession** - *derived from* - **Parent sample accession**
+
+For example, in the first linked dataset, the `EMC study <https://www.ebi.ac.uk/about/news/updates-from-data-resources/pathogens-portal-linked-dataset/>`_,
+the BioSamples relationship is as follows:
+
+**[T/B-Cell/Antibody profile/ENA viral sample accession]** - *derived from* - **[Top level patient sample accession]**
+
+A JSON file curation object (see example below) containing the relationship attribute with the source and target sample
+can be created and submitted via curl to the `BioSamples API <https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_curation_object>`_)
+
+JSON curation:
+
+.. code-block:: JSON
+
+   {
+     "curation" : {
+      "relationshipsPre" : [ ],
+       "relationshipsPost" : [ {
+         "source" : "SAMFAKE123456",
+         "type" : "DERIVED_FROM",
+         "target" : "SAMFAKE7654321"
+       } ],
+       "hash" : "09a5a9cddbea9f5bb6302b86b922c408abc92b8b10c78f0662ac7e41fd44e91f"
+     },
+    "domain" : null,
+    "webinSubmissionAccountId" : "WEBIN-12345",
+     "created" : "2023-07-17T12:19:33.056356Z",
+     "hash" : "d1f611ec2c8caf3d9f58fa40227ea60ebb5fc00eda06338fb81db7d987a6fa63"
+   }
+
+..
+
+There is also a `python script<https://www.ebi.ac.uk/biosamples/docs/references/api/submit#_submit_curation_object>`_
+which can be used to create the sample relationships. Please contact ena-path-collabs@ebi.ac.uk for technical support
```

Comment:
I see you have not linked the python script here, but I was wondering if we can hold off on sharing that just yet anyway - as it's only really been tested on one use-case. 
We have another 2 linking use-cases coming up that I am actively working on, so would like to test the script on at least one of those before sharing

Reply:
this section has been removed